deployments/azure-webapps-php.yml

```diff
@@ -0,0 +1,80 @@
+# This workflow will build and push a node.js application to an Azure Web App when a commit is pushed to your default branch.
+#
+# This workflow assumes you have already created the target Azure App Service web app.
+# For instructions see https://docs.microsoft.com/en-us/azure/app-service/quickstart-nodejs?tabs=linux&pivots=development-environment-cli
```

Comment:
Same here--should this reference PHP instead of Node?

Reply:
Thanks, copy/paste error. Fixed it